packages/kotti-ui/source/kotti-popover/KtPopoverIconTextItem.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+	<div
+		class="kt-popover-item"
+		:class="{
+			'kt-popover-item--is-disabled': isDisabled,
+		}"
+		v-on="$listeners"
+	>
+		<i v-show="icon" class="yoco" v-text="icon" />
+		<div>{{ label }}</div>
+	</div>
+</template>
+
+<script>
+export default {
+	name: 'KtPopoverIconTextItem',
+	props: {
+		icon: {
+			type: String,
+			default: null,
+		},
+		isDisabled: {
+			type: Boolean,
+		},
+		label: {
+			type: String,
```

Comment:
same but you want to fallback to `null` here, not `''` b/c like I said, the user app might want to have `''` for whatever reason
and maybe even v-if on https://github.com/3YOURMIND/kotti/pull/363/files#diff-dbaf4ef6086fcffffa4d4c846ad5bc90cf8fc93db9a38dbc8f299f197c498ec0R10 that it's not `null`